guides/v2.2/coding-standards/technical-guidelines.md

```diff
@@ -98,6 +100,11 @@ class Config
 {% endcollapsible %}
 ---
 
+2.2.2. Factories SHOULD be used for object instantiation instead of `new` keyword. An object SHOULD be replaceable for testing or extensibility purposes.
+Exception: [DTOs](https://en.wikipedia.org/wiki/Data_transfer_object). There is no behavior in DTOs, so there is no reason for its replaceability.
+Tests can create real DTOs for stubs. 
+Data interfaces,  Exceptions and `Zend_Db_Expr` are examples of DTOs.
```

Comment:
Would be great to add code samples where we can't test or extend some class and how using factories will help to solve this issue

Reply:
Maybe having a blog post would be helpful? I'm afraid adding more explanations here would make the document grow and hard to read.

Reply:
> Maybe having a blog post would be helpful? I'm afraid adding more explanations here would make the document grow and hard to read.

Good idea + add link to that post there